papaparse.js

```diff
@@ -1290,8 +1295,8 @@
 		}
 	}
 
-	/** https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions */
-	function escapeRegExp(string) {
+	function escapeRegExp(string)
```

Comment:
Why changing this lines?

Reply:
Consistency with the coding style of the rest of the functions.

Reply:
I think the comment referencing the source material makes sense to keep.